lib/carelink/suspend.js

```diff
@@ -33,23 +33,68 @@ module.exports = function (timezone) {
               case 'null':
                 return null;
               case 'user_suspend':
-                return { status: 'suspended', reason: 'manual' };
+                return {
+                  status: 'suspended',
+                  reason: {
+                    suspended: 'manual'
+                  }
+                };
               case 'low_suspend_mode_1':
               case 'alarm_suspend':
               case 'low_suspend_no_response':
               case 'low_suspend_user_selected':
-                return { status: 'suspended', reason: 'low_glucose' };
-
+                return {
+                  status: 'suspended',
+                  reason: {
+                    suspended: 'automatic'
+                  },
+                  payload: {
+                    cause: 'low_glucose',
+                    code: reason
```

Comment:
`reason` is the raw CareLink string, so I named the attr `code`. Does that make it clearer?